engine/src/main/scala/cromwell/engine/workflow/tokens/JobExecutionTokenDispenserActor.scala

```diff
@@ -126,6 +118,7 @@ class JobExecutionTokenDispenserActor(override val serviceRegistryActor: ActorRe
       case LeasedActor(queuePlaceholder, lease) =>
         log.error(s"Actor ${queuePlaceholder.actor.path} requested a job execution token more than once. This situation should have been impossible.")
         queuePlaceholder.actor ! JobExecutionTokenDispensed
+        // If there are going to be > 1 actors mapping to this lease why is the lease being released?
         lease.release()
```

Comment:
Before merging, can we also get an answer to this question and update the comment?